packages/react/src/components/form/authenticating/otp.tsx

```diff
@@ -50,12 +50,13 @@ export const OTPState = ({ flowState, performLogin }: Props) => {
   >("initial");
   const submitInputRef = useRef<HTMLInputElement>(null);
 
-  const factor = flowState.context.config.factor;
+  const { factor, handle } = flowState.context.config;
   const hasRetried = flowState.context.attempt > 1;
   const { title, message } = getAuthenticatingMessage(factor, {
     isSubmitting: formState === "submitting",
     hasRetried,
   });
+  const tokens = getTokensFromHandle(handle);
```

Comment:
It seems strange that the message tokens are returned separately from the message. Would it make sense if `getAuthenticating` message returns both the message and the related tokens?